llvm/unittests/Support/KnownBitsTest.cpp

```diff
@@ -367,9 +315,9 @@ TEST(KnownBitsTest, BinaryExhaustive) {
   testBinaryOpExhaustive(KnownBits::smax, APIntOps::smax);
   testBinaryOpExhaustive(KnownBits::smin, APIntOps::smin);
   testBinaryOpExhaustive(KnownBits::abdu, APIntOps::abdu,
-                         checkCorrectnessOnlyBinary);
+                         checkOptimalityBinary);
```

Comment:
This is default, you can just remove the argument.

Reply:
Done, thanks.